ergotree-ir/src/sigma_protocol/dlog_group.rs

```diff
@@ -190,6 +215,34 @@ impl SigmaSerializable for EcPoint {
     }
 }
 
+impl ScorexSerializable for EcPoint {
```

Comment:
There is no sense to have an identical `SigmaSerializable` impl. Could you please kill it and use `ScorexSerializable` methods at call sites?

Reply:
It's not currently possible to replace the methods with `ScorexSerializable` equivalents because `ScorexSerializationError` and `SigmaSerializationError` enums differ slightly:
  - `ScorexSerializationError::TryFrom(_)` variant has no equivalent in `SigmaSerializationError`
  - `SigmaSerializationError::UnexpectedValue(TryExtractFromError)` has no equivalent in `ScorexSerializationError`

Similarly there are differences between `ScorexParsingError` and `SigmaParsingError`. Now the set of variants of `ScorexParsingError` is *almost* a subset of the variants of `SigmaParsingError`.

Does it make sense to add in the missing variants to `Sigma[Parsing|Serialization]Error` and then `impl From<ScorexParsingError> for SigmaParsingError` etc? Then the implementation of `SigmaSerializable` can be defined in terms of `ScorexSerializable` for `EcPoint`. Probably simpler to just go with the copy-paste impl I already have. What do you think?

Reply:
Good call! I stumbled upon the same issue in my REST-SPV branch and came up with the same idea - https://github.com/ergoplatform/sigma-rust/blob/5d19cbfdce9fc4830221c07790e97bee03199e78/ergotree-ir/src/serialization/serializable.rs#L35-L37

Reply:
Thanks, I'll do the same here.